docs/admin/multiple-schedulers.md

```diff
@@ -0,0 +1,236 @@
+$ gcloud docker push gcr.io/my-gcp-project/my-kube-scheduler:1.0
+```
+
+### 2. Define a Kubernetes deployment for the scheduler
+
+Now that we have our scheduler in a container image already, we can just create a pod
+config for it and run it in a Kubernetes cluster. But instead of just running a pod in
+the cluster, let's use a [Deployment](http://kubernetes.io/docs/user-guide/deployments/)
+for this example. A [Deployment](http://kubernetes.io/docs/user-guide/deployments/)
+manages a [Replica Set](http://kubernetes.io/docs/user-guide/replicasets/) which in turn
+manages the pods, thereby making the scheduler resilient to failures. Here is the
+deployment config. Save it as `my-scheduler.yaml`:
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    component: scheduler
+    tier: control-plane
+  name: my-scheduler
+  namespace: kube-system
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels: 
+        component: scheduler
+        tier: control-plane
+        version: second
+    spec:
+      containers:
+      - command: [/usr/local/bin/kube-scheduler, --address=0.0.0.0,
+                  --scheduler-name=my-scheduler]
+        image: gcr.io/my-gcp-project/my-kube-scheduler:1.0
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 10251
+          initialDelaySeconds: 15
+        name: kube-second-scheduler
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 10251
+        resources:
+          requests:
+            cpu: '0.1'
+        securityContext:
+          privileged: false
+        volumeMounts: []
+      hostNetwork: false
+      hostPID: false
+      volumes: []
+```
+
+An important thing to note here is that the name of the scheduler specified as an
+argument to the scheduler command in the container spec should be unique. This is the name that is matched against the value of the optional `scheduler.alpha.kubernetes.io/name` annotation on pods, to determine whether this scheduler is responsible for scheduling a particular pod.
+
+Please see the
+[kube-scheduler documentation](http://kubernetes.io/docs/admin/kube-scheduler/) for
+detailed description of other command line arguments.
+
+### 3. Run the second scheduler in the cluster
+
+In order to run your scheduler in a Kubernetes cluster, just create the deployment
+specified in the config above in a Kubernetes cluster:
+
+```shell
+$ kubectl create -f my-scheduler.yaml
+```
+
+Verify that the scheduler pod is running:
+
+```shell
+$ kubectl get pods --namespace=kube-system
+NAME                                           READY     STATUS    RESTARTS   AGE
+....
+my-scheduler-lnf4s-4744f                       1/1       Running   0          2m
+...
+```
+
+You should see a "Running" kube-scheduler pod. It is also important to note that the
+default scheduler managed by Kubernetes runs in a container but not in a pod, so you
```

Comment:
@davidopp I made this correction in a comment earlier and promised to fix it. But evidently, I haven't. Sorry about that. Fixed it now.